app/views/layouts/_proxy_notice.html.erb

```diff
@@ -0,0 +1,18 @@
+<div id="proxy-notice">
+  <div class="content">
+    <p class="english">Notice: (1) You are using a proxy site that is not affiliated with AO3. (2) The entity that set up the proxy site can see whatever data you submit to the AO3 through the proxy site, including your IP address and the content of comments.</p>
```

Comment:
I wasn't sure if we should make the banner text translatable. Putting the English in the en.yml locale file made sense, but not so much putting the Chinese there.

Reply:
We can put both in locale files, because it's possible to force a specific language for one `I18n.t` call.

https://guides.rubyonrails.org/i18n.html#setting-and-passing-a-locale